<?xml version="1.0" ?>
<launch>


	<arg name="ft_sensor_topic"		value="/ft_classifier"/>
	<arg name="virtual_sensor_topic" 	value="/virtual_classifier"/>

	<arg name="action_topic" 		value=	"lwr_peg_link"/>
	<arg name="peg_link_name" 		value=	"lwr_peg_link"/>



	<!-- launch FT sensor -->
	<include file="$(find netft_rdt_driver)/launch/ft_sensor.launch">
		<arg  name="rate" 	 value="100"/>	
		<arg  name="frame_id"    value="lwr_peg_link"/>
		<arg  name="alpha"       value="0.99"/>
		<arg  name="rot"	 value="0.7"/>
		<arg  name="scale_y" 	 value="-1"/>
	</include>

	<!-- launch python ft sensor -->
	<include file="$(find ft_sensor_python)/launch/ft_sensor.launch"/>

	<!-- launch peg sensor -->
	<include file="$(find peg_sensor)/launch/peg_sensor.launch">
		<arg name="y_topic" value="$(arg virtual_sensor_topic)"/>
	</include>	

	<!-- launch particle filter -->
	<!--include file="$(find peg_filter)/launch/peg_particle_filter.launch">
		<arg name="ft_sensor_topic" 		value="$(arg ft_sensor_topic)"/>
		<arg name="virtual_sensor_topic" 	value="$(arg virtual_sensor_topic)"/>
		<arg name="action_topic" 		value="$(arg action_topic)"/>
		<arg name="peg_link_name" 		value="$(arg peg_link_name)"/>
		<arg name="sensor_type" 		value="$(arg sensor_type)"/>
	</include-->

</launch>
